index.html

```diff
@@ -53,15 +59,958 @@
+                        </div>
+                        <div class="clearfix">
+                            <p>"Cobra Kai" Season 4 has been confirmed by Netflix, and production has been reported to be...
+                            </p>
+                        </div>
+                    </article>
+                    <article class="border-top border-2 mb-3 py-2">
+                        <div class="float-start">
+                            <a href="#"><img src="image/middle.png" class="img-fluid width me-3 pt-2" alt="..."></a>
+                        </div>
+                        <div>
+                            <h5><a href="#" class="link-dark nav-link p-0 mt-2">'Cobra Kai' Season 4 Expected to Start Shooting Today</a></h5>
+                        </div>
+                        <div class="clearfix">
+                            <p>"Cobra Kai" Season 4 has been confirmed by Netflix, and production has been reported to be...
+                            </p>
+                        </div>
+                    </article>
+                </section>
+                <div class="gray w-100 d-md-none"> <br></div>
+            </div>
+            <div class="col-md-7 col-lg-5">
+                <!-- TOP STORY -->
+                <section class="d-none d-md-inline">
+                    <a href="#" class="text-decoration-none"><h5 class="text-secondary fw-bold">TOP STORY</h5></a>
+                    <article class="mt-3">
+                        <div class="position-relative">
+                           <a href="#"><img src="image/middle.png" class="img-fluid topwidth" alt="..."></a>
+                            <div class="position-absolute bottom-0 start-10 bg-light1"><span
+                                    class="w-100 text-center fw-bold fs-4">&nbsp; U.S &nbsp; &nbsp;</span></div>
+                        </div>
+                        <div>
+                            <h4><a href="#" class="link-dark nav-link p-0 mt-2">Joe Biden's First Executive Actions Undo
+                                    Many of Donald Trump's</a></h4>
+                        </div>
+                        <div>
+                            <p>Actions related to the COVID-19 crisis, climate change and racial equality are to be
+                                among the first taken as he begins his tenure.</p>
+                        </div>
+                    </article>
+                </section>
+                <!-- MY TURN -->
+                <section>
+                    <a href="#" class="text-decoration-none"><h5 class="text-secondary fw-bold mt-3 ms-md-0">MY TURN</h5></a>
+                    <div class="d-flex flex-column flex-md-row font-medium">
+                        <div>
+                            <a href="#"><img src="image/middle.png" class="img-fluid pe-md-2" alt="..."></a>
+                            <h5 class="text-secondary my-4 my-md-2">'I Married My Best Friend to Avoid Inheritance Tax' </h5>
```

Comment:
- [ ] All the titles should have a link and more font-weight. (Each title uses a hx tag and has a link) - Check the comment below. As you can see on the original website, this and all the other similar headers are links. So please add the appropriate anchor tag.